boa/src/environment/mod.rs

```diff
@@ -1,8 +1,3 @@
-//! Environment handling, lexical, object, function and declaritive records
+//! Environment handling, lexical, object, function and declarative records
 
-pub mod declarative_environment_record;
-pub mod environment_record_trait;
-pub mod function_environment_record;
-pub mod global_environment_record;
 pub mod lexical_environment;
-pub mod object_environment_record;
```

Comment:
Maybe most of this can be removed, and everything that's left should come to a single file.